base/settings/R/check.all.settings.R

```diff
@@ -1024,8 +1024,9 @@ check.database.settings <- function(settings) {
       # should runs be written to database
       if (is.null(settings$database$bety$write)) {
         PEcAn.logger::logger.info(
-          "Writing all runs/configurations to database.")
-        settings$database$bety$write <- TRUE
+          "database$bety$write not set.",
+          "Will not write any runs/configurations to database.")
+        settings$database$bety$write <- FALSE
```

Comment:
Here's the change in existing behavior: when `settings$database$bety$write` was undefined it used to default to TRUE, but defaulting to FALSE means DB-free runs can delete the entire settings$database section.

Reply:
Turns out we have existing test coverage for this! I'll update the test with the new behavior.